src/commands.ts

```diff
@@ -248,6 +248,147 @@ export async function createNewProject(ctx: WorkspaceContext): Promise<void> {
     }
 }
 
+/**
+ * Prompts the user to input project details and then executes `swift package init`
+ * to create the project.
+ */
+export async function browsePackageIndex(ctx: WorkspaceContext): Promise<void> {
+    // The context key `swift.browsePackageIndex` only works if the extension has been
+    // activated. As such, we also have to allow this command to run when no workspace is
+    // active. Show an error to the user if the command is unavailable.
+    if (!ctx.toolchain.swiftVersion.isGreaterThanOrEqual(new Version(6, 0, 0))) {
+        vscode.window.showErrorMessage(
+            "Creating a new swift project is only available starting in swift version 6.0.0."
```

Comment:
Interesting error? If you do want to gate access to a feature you should use a context key to disable the command

Reply:
Fixed the wording of the text and added a context key to block the command.